analytics.go

```diff
@@ -57,7 +57,7 @@ type client struct {
 
 	// This HTTP client is used to send requests to the backend, it uses the
 	// HTTP transport provided in the configuration.
-	http http.Client
+	http *http.Client
 }
```

Comment:
What's the reason you changed this to a pointer?
The http.Client object's lifetime is bound to the analytics.client's, it feels like it's giving more work to the GC for no reason.